.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,227 @@
+name: GEOS CI 
+on: pull_request
```

Comment:
I think we'll be able to trigger specific actions based on PR labels, that will be helpful.

Reply:
> I think we'll be able to trigger specific actions based on PR labels, that will be helpful.

Yes, Github Actions has alot of control over triggering workflow runs:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

In a future PR we can really customize/limit what triggers a CI run.